jetty-util/src/main/java/org/eclipse/jetty/util/ssl/KeyStoreScanner.java

```diff
@@ -88,11 +78,23 @@ public KeyStoreScanner(SslContextFactory sslContextFactory, boolean followLinks)
         addBean(_scanner);
     }
 
+    private Path getRealKeyStorePath()
+    {
+        try
+        {
+            return keystoreFile.toPath().toRealPath();
```

Comment:
You could assign `keystoreFile.toPath()` to a variable and prevent doing this twice in the case of a failure.
(not that this matters that much as its only for debugging)